laws/src/main/scala/cats/laws/InvariantMonoidalLaws.scala

```diff
@@ -4,16 +4,16 @@ package laws
 /**
  * Laws that must be obeyed by any `cats.InvariantMonoidal`.
  */
-trait InvariantMonoidalLaws[F[_]] extends InvariantLaws[F] with SemigroupalLaws[F] {
+trait InvariantMonoidalLaws[F[_]] extends InvariantSemigroupalLaws[F] {
   override implicit def F: InvariantMonoidal[F]
   import cats.syntax.semigroupal._
   import cats.syntax.invariant._
 
   def invariantMonoidalLeftIdentity[A, B](fa: F[A], b: B): IsEq[F[A]] =
-    F.pure(b).product(fa).imap(_._2)(a => (b, a)) <-> fa
+    F.point(b).product(fa).imap(_._2)(a => (b, a)) <-> fa
```

Comment:
We can express these laws in `unit` right?  like
```scala
def invariantMonoidalLeftIdentity[A, B](fa: F[A]): IsEq[F[A]] =
 F.unit.product(fa).imap(_._2)(a => ((), a)) <-> fa		 
```
Then we can add a `point` - `unit` consistency law. WDYT?

Reply:
Agreed!